contracts/SpokePool.sol

```diff
@@ -141,9 +146,12 @@ abstract contract SpokePool is
     // token for the input token. By using this magic value, off-chain validators do not have to keep
     // this event in their lookback window when querying for expired deposts.
     uint32 public constant INFINITE_FILL_DEADLINE = type(uint32).max;
-    /****************************************
```

Comment:
you might not have meant to modify bits like this?

Reply:
No, I have some issues with the formatter, I will try to fix my formatter or manually revert this changes